guides/concepts/list-of-operations.md

```diff
@@ -77,19 +77,65 @@ Possible errors:
+
+A Path Payment Strict Send allows a user to specify the *amount of the asset to send*. The amount received will vary based on offers in the order books.  If you would like to instead specify the amount received, use the [Path Payment Strict Receive](#path-payment-strict-receive) operation.  
+
+A few things to note:
+* path payments don't allow intermediate offers to be from the source account as this would yield a worse exchange rate. You'll need to either split the path payment into two smaller path payments, or ensure that the source account's offers are not at the top of the order book.
+* balances are settled at the very end of the operation
+   * this is especially important when `(Destination, Destination Asset) == (Source, Send Asset)` as this provides a functionality equivalent to getting a no interest loan for the duration of the operation.
+
+Threshold: Medium
+
+Result: `PathPaymentStrictSendResult`
+
+Parameters:
+
+|Parameters| Type| Description|
+| --- | --- | --- |
+|Send asset| asset| The asset deducted from the sender's account.|
+|Send amount| integer| The amount of `send asset` to deduct (excluding fees).|
+|Destination| account ID| Account ID of the recipient.|
+|Destination asset| asset| The asset the destination account receives.|
+|Destination min| integer| The minimum amount of `destination asset` the destination account can receive.|
+|Path| list of assets| The assets (other than `send asset` and `destination asset`) involved in the offers the path takes. For example, if you can only find a path from USD to EUR through XLM and BTC, the path would be USD -> XLM -> BTC -> EUR and the `path` field would contain XLM and BTC.|
+
+Possible errors:
+
+| Error | Code | Description |
+| ----- | ---- | ------|
+|PATH_PAYMENT_STRICT_SEND_MALFORMED| -1| The input to this path payment is invalid.|
+|PATH_PAYMENT_STRICT_SEND_UNDERFUNDED| -2| The source account (sender) does not have enough funds to send and still satisfy its selling liabilities. Note that if sending XLM then the sender must additionally maintain its minimum XLM reserve.|
+|PATH_PAYMENT_STRICT_SEND_SRC_NO_TRUST| -3| The source account does not trust the issuer of the asset it is trying to send.|
+|PATH_PAYMENT_STRICT_SEND_SRC_NOT_AUTHORIZED| -4| The source account is not authorized to send this payment. |
+|PATH_PAYMENT_STRICT_SEND_NO_DESTINATION| -5| The receiving account does not exist. |
+|PATH_PAYMENT_STRICT_SEND_NO_TRUST| -6| The receiver does not trust the issuer of the asset being sent. For more, see the [assets doc](./assets.md).|
+|PATH_PAYMENT_STRICT_SEND_NOT_AUTHORIZED| -7| The destination account is not authorized by the asset's issuer to hold the asset. |
+|PATH_PAYMENT_STRICT_SEND_LINE_FULL| -8| The destination account (receiver) does not have sufficient limits to receive `destination amount` and still satisfy its buying liabilities.|
```

Comment:
I don't know why this line needs the `(receiver)` clarification if the others don't.
```suggestion
|PATH_PAYMENT_STRICT_SEND_LINE_FULL| -8| The destination account does not have sufficient limits to receive `destination amount` and still satisfy its buying liabilities.|
```